Cargo.toml

```diff
@@ -17,7 +17,7 @@ amplify = "4.0.0"
 io-reactor = { version = "0.2.0", optional = true }
 cyphernet = { version = "0.3.0", features = ["ed25519", "pem", "multibase", "noise_sha2", "noise_x25519", "noise-framework", "mixnets", "dns"] }
 mio = { version = "0.8.6", optional = true }
-socket2 = { version = "0.5.2", optional = true }
+socket2 = { version = "0.5.2", features = ["all"] }
```

Comment:
Since `all` can be extended with minor release, I think it is better to commit here to a specific set of features we need.

Reply:
`all` is `socket2`'s only feature. I can't commit to anything more fine-grained than that.